src/Symfony/Component/VarDumper/Caster/ConstStub.php

```diff
@@ -30,4 +30,27 @@ public function __toString()
     {
         return (string) $this->value;
     }
+
+    /**
+     * Creates ConstStub from bitfield flag.
+     *
+     * @param int    $value  The value of bitfield flag
+     * @param string $prefix The prefix that filters flags
+     *
+     * @return self
+     */
+    public static function fromFlag($value, $prefix)
+    {
+        $constants = get_defined_constants();
```

Comment:
Yes, we can add a ``$from`` param that does something like:
```php
$constants = null === $from ? get_defined_constants() : (new \ReflectionClass($from))->getConstants();
```
But for now, it's not yet useful.